semconvgen/README.md

```diff
@@ -0,0 +1,28 @@
+# Go Semantic Convention Generator
+
+This tool is designed to generate constants in a semantic convention package for the Go API and the collector.
+It may be used by other systems, but it's primary function beyond invoking the template processor is to ensure that
+generated identifiers conform to Go's naming idiom, particularly with respect to initialisms and acronyms.
+Other users may be served just as well by using the template processor directly.
+
+## Installation
+
+```shell
+$ go get go.opentelemetry.io/build-tools/semconvgen
+```
+## Usage
+
+```shell
+$ semconvgen -i <path to spec YAML> -t <path to template> -o <path to output>
```

Comment:
semconvgen is already the name of the docker container for the other semantic convention generator, so I recommend a different name.